shared-ui/src/main/res/layout/confirm_image_sheet_layout.xml

```diff
@@ -73,6 +74,10 @@
         app:layout_constraintTop_toBottomOf="@id/standard_sheet_content"
         tools:text="Confirm "
         android:layout_marginTop="40dp"
+        android:padding="@dimen/text_button_padding_size"
+        android:background="@drawable/clickable_text_background"
+        android:clickable="true"
+        android:focusable="true"
```

Comment:
What is the purpose of adding `focusable`into `TextView`here, do we need to set focus on the `TextView`when we show the `BottomSheet`?

Reply:
Probably not. Just added it since is clickable and shows a UI response to click events.